.github/workflows/daily.yml

```diff
@@ -143,6 +143,31 @@ jobs:
         if: ${{ failure() }}
       - uses: ./.github/actions/build-teardown
 
+  prover-inconsistency-test:
+    runs-on: ubuntu-latest-xl
+    container:
+      image: diem/build_environment:main
+    env:
+      MESSAGE_PAYLOAD_FILE: /tmp/message
+    strategy:
+      fail-fast: false
+      matrix:
+        release-branch: [release-1.1]
```

Comment:
```suggestion
        target-branch: [main]
```

Reply:
You might want to build both?   Or just stick with main.

Reply:
I keep both `main` and `release-1.1`.